websites/T/Tellingtone/presence.ts

```diff
@@ -0,0 +1,97 @@
+const presence = new Presence({ clientId: "1006201873985961984" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			viewHome: "general.viewHome",
+			paused: "general.paused",
+			playing: "general.playing",
+			episode: "general.episode",
+			viewPage: "general.viewPage",
+			buttonViewSeries: "general.buttonViewSeries",
+			currentlyListening: "general.currentlyListening",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+
+let strings: Awaited<ReturnType<typeof getStrings>>,
+	oldLang: string = null;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href } = document.location,
+		pathSplit = pathname.split("/"),
+		[newLang, time, showCover] = await Promise.all([
+			presence.getSetting<string>("lang").catch(() => "en"),
+			presence.getSetting<boolean>("time"),
+			presence.getSetting<boolean>("cover"),
+		]);
+
+	if (oldLang !== newLang || !strings) {
+		oldLang = newLang;
+		strings = await getStrings();
+	}
+
+	switch (pathSplit[1]) {
+		case "":
+			presenceData.details = strings.viewHome;
+			break;
+		case "shop":
+		case "blog":
+			presenceData.details = `Viewing ${pathSplit[1]}`;
+			break;
+		case "series":
+			presenceData.details = strings.viewPage;
+			presenceData.state =
+				document.querySelector<HTMLHeadingElement>("div.head > h1").textContent;
+			presenceData.largeImageKey =
+				document.querySelector<HTMLImageElement>("div.head > img").src;
+
+			presenceData.buttons = [{ label: strings.buttonViewSeries, url: href }];
+	}
+	if (document.querySelector("div#Player")) {
+		presenceData.details = strings.currentlyListening.replace(
```

Comment:
if you arent using {0} you should still probably replace it to an empty string, otherwise {0} will show up in the presencedata

Reply:
Should be finally okay